documentation/API-design-guidelines.md

```diff
@@ -1672,18 +1694,27 @@ The rationale for using this alternate pattern should be explicitly provided
 (e.g. the notification source for each of the supported events may be completely different,
 in which case, separating the implementations is beneficial). 
 
+##### Rules for subscriptions data minimization 
```

Comment:
A placeholder added as comment to consider the generic advice hint mentioned here for event structure design :) https://github.com/camaraproject/Commonalities/issues/295#issuecomment-2418788007

Reply:
I've added the sentence: Application of data minimization design must be considered by the API working group for event structure definition.